internal/cmd/common.go

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (C) 2021 Canonical Ltd
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0'
+ */
+
+package cmd
+
+import (
+	"github.com/edgexfoundry/edgex-cli/internal/config"
+	"github.com/edgexfoundry/edgex-cli/internal/format"
+	"github.com/spf13/cobra"
+)
+
+var verbose, metadata, data, command, notification, scheduler, json bool
+
+func getSelectedService() string {
+	if metadata {
+		return "core-metadata"
+	} else if data {
+		return "core-data"
+	} else if command {
+		return "core-command"
+	} else if notification {
+		return "core-notification"
+	} else if scheduler {
+		return "core-scheduler"
+	} else {
+		return ""
+	}
+}
+
+func dataService(cmd *cobra.Command, api string, name string) error {
+	s := config.GetCoreService("core-data")
+	value, _, _, err := format.GetServiceValue(s, api, name)
+	if err != nil {
+		cmd.Println(err.Error())
+	} else {
+		cmd.Println(value)
+	}
+	return nil
```

Comment:
Why have return value if always return `nil`?

Reply:
This method should not even be there and I've now removed it.